store/items.ts

```diff
@@ -0,0 +1,71 @@
+import Vue from 'vue';
+import { BaseItemDto } from '@jellyfin/client-axios';
+import { ActionTree, GetterTree, MutationTree } from 'vuex';
+import map from 'lodash/map';
+import some from 'lodash/some';
+import forEach from 'lodash/forEach';
+
+export interface ItemsState {
+  byId: Record<string, BaseItemDto>;
+  allIds: string[];
```

Comment:
What's the difference between this and Object.keys(byId)? Why do we need this specifically?

Reply:
This is inspired by Redux way of doing (cc @MrTimscampi). It allows both to save time instead of doing Object.keys and allows to store order.

Reply:
More or less, Object.keys() makes a copy of the object, which is expensive. Objects also don't guarantee any sort of order is kept.

The reasoning is that doing this helps us keep the order, if we need it, and avoids doing potentially expensive copies every time we need to check if something is in the store.